src/Resources/Resources/help/Set-AzPolicyAssignment.md

```diff
@@ -209,6 +238,36 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
+### -PolicyParameter
```

Comment:
@mentat9 you be able to add examples using the new `-PolicyParameter` and `-PolicyParameterObject` parameters?

Reply:
Done.